src/pages/send/send.ts

```diff
@@ -52,9 +55,11 @@ export class SendPage {
   ionViewWillEnter() {
     this.walletsBtc = this.profileProvider.getWallets({ coin: 'btc' });
     this.walletsPolis = this.profileProvider.getWallets({ coin: 'polis' });
+    this.walletsDash = this.profileProvider.getWallets({ coin: 'dash' });
     this.hasBtcWallets = !(_.isEmpty(this.walletsBtc));
-    this.hasPolisWallets = !(_.isEmpty(this.walletsPolis));
+    this.hasDashWallets = !(_.isEmpty(this.walletsDash));
```

Comment:
Why replacing this.hasPolisWallets = !(_.isEmpty(this.walletsPolis)); by Dash and not add in another line ?